src/lib/components/drip-lists-section/drip-lists-section.svelte

```diff
@@ -107,12 +125,31 @@
                   modal.show(CreateDripListStepper, undefined, {
                     skipWalletConnect: true,
                     isModal: true,
-                  })}>Create a new Drip List</Button
-              >
+                  })}
+                >Create a new Drip List
+              </Button>
             </div>
           </div>
         </div>
       {/if}
     </div>
+
+    <div>
+      {#if isOwner}
+        <VisibilityToggle bind:showHidden hiddenItemsCount={hiddenListsCount} />
+      {/if}
+    </div>
+
+    <div
+      class="grid gap-6 grid-cols-1 padding pt-px {hiddenDripListsAndVotingRounds.length > 0
```

Comment:
Wish I was this fluent with the helper classes 🥲

Reply:
all copy-paste magic  😇